src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -107,37 +120,23 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
         )
       );
 
-      //todo tools functionality needs to be added to toolbar
-      testIf(theme !== 'refresh-toolbar')(
-        'focuses tools panel closed button when it is opened using keyboard and caused split panel to change position',
+      test(
+        'focuses tools panel closed button when it is opened and caused split panel to change position',
         setupTest(
           async page => {
-            await page.setWindowSize({ width: 1000, height: 800 });
+            await page.setWindowSize({ width: 878, height: 800 });
```

Comment:
There is open navigation on that page. On such small screen size, the split panel is in bottom position because of that anyway. Are you sure the test is still authentic in that case?

Reply:
This was interesting. When I tested it locally, these sizes seemed to work fine. But just tested again and even 1000 is too small for VR. Increased the width and added assertions to ensure the panel is in the proper positions.

Reply:
Update, I think it had to do with loading the page on mobile size vs desktop. Also added a reset for mobile, where nav is closed at page load.